Taskfile.yml

```diff
@@ -169,7 +169,7 @@ tasks:
         "{{.OUTPUT_DIR}}/var/www/log_viewer_webui/server/"
       - |-
         cd "{{.OUTPUT_DIR}}/var/www/log_viewer_webui/server"
-        PATH="{{.G_NODEJS_22_BIN_DIR}}":$PATH npm clean-install
+        PATH="{{.G_NODEJS_22_BIN_DIR}}":$PATH npm install
```

Comment:
Why not `clean-install`?

Reply:
Sorry, i meant not to make everything "clean-install" in this PR since not all are directly related to lint, but reverted the wrong line.